src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -166,4 +168,43 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String dateTime} into an {@code DateTime}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code dateTime} is invalid.
+     */
+    public static DateTime parseDateTime(String dateTime) throws ParseException {
+        requireNonNull(dateTime);
+        String trimmedDateTime = dateTime.trim();
+        return new DateTime(trimmedDateTime);
+    }
+
+    /**
+     * Parses a {@code String condition} into an {@code Condition}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code condition} is invalid.
+     */
+    public static Condition parseCondition(String condition) {
+        requireNonNull(condition);
+        String trimmedCondition = condition.trim();
+
+        return new Condition(trimmedCondition);
+    }
```

Comment:
Should add checks on whether they are valid DateTime and Condition, can take reference from rest of code.

Reply:
Fixed